CMakeLists.txt

```diff
@@ -1,4 +1,5 @@
-cmake_minimum_required(VERSION 3.13)  # for add_link_options
+cmake_minimum_required(VERSION 3.20)  # for CMP0115
```

Comment:
Let's not bump CMake version - this will break CMake builds on many distros that have not yet updated

Reply:
Hm, without this CMake will not let me refer to build-info.cpp if there is also a file called build-info.cpp.in, as it assumes I forgot the extension. I will have to find a different way to do this.

Reply:
Actually, it seems like this policy change wasn't necessary after all. I must have seen the warning before I got the build info dependency fully working.